src/pages/outlier-detection/OutlierDetection.js

```diff
@@ -277,11 +275,6 @@ class OutlierDetection extends Page {
                         </FontIcon>
                     </IconButton>
                     <h1>{i18n.t('Outlier Detection')}</h1>
-                    <PageHelper
```

Comment:
There is no documentation for this page and so I have removed the docs link component.

I've filed https://jira.dhis2.org/browse/DOCDHIS2-92 to track this.